manticore/manticore.py

```diff
@@ -294,37 +268,6 @@ def maxstorage(self):
     def maxstorage(self, max_storage):
         self._maxstorage = max_storage
 
-    @property
-    def verbosity(self):
```

Comment:
Hm, we also never did reach a clear decision on whether to make this interface global, as opposed to associated with the Manticore class (@feliam mentioned it here https://github.com/trailofbits/manticore/issues/431#issuecomment-320025228), although to be honest, I'm starting to reconsider whether we should. Between the options of

```
from manticore import Manticore
m = Manticore('/bin/ls')
m.verbosity = 3
m.run()
```
and
```
from manticore import Manticore, set_verbosity
m = Manticore('/bin/ls')
set_verbosity(3)
m.run()
```
I do prefer the former, mostly because I don't like the namespace clutter that `set_verbosity` adds. Thoughts @trailofbits/manticore?